### ISSUE TITLE
Install cernlib error (MacOS 13.6, M2 chip)

### ISSUE THREAD

**Issue by Xiu-Lei** [davidchall/homebrew-hep#325]
### Verification

- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

I am trying to install cernlib on MacOS 13.6, with M2 apple chip. But it does not work, and I get the following error... 

### What happened (include all command output)?

 brew install cernlib 
==> Downloading https://formulae.brew.sh/api/formula.jws.json
##O#-  #                                                                                                               
==> Downloading https://formulae.brew.sh/api/cask.jws.json

==> Fetching davidchall/hep/cernlib
==> Downloading https://gist.githubusercontent.com/andriish/0f0142dcb5e9717d6cfc9235f2745ec3/raw/c7ba93ebb2ea2471a3f315b
################################################################################################################# 100.0%
==> Downloading https://cernlib.web.cern.ch/cernlib/download/2022_source/tar/cernlib-2022.11.08.0-free.tar.gz
################################################################################################################# 100.0%
==> Installing cernlib from davidchall/hep
==> Patching
==> Applying patch-cernlib20221108-1.txt
patching file CMakeLists.txt
patching file 'mathlib/CMakeLists.txt'
Error: An exception occurred within a child process:
  NoMethodError: undefined method `version_suffix' for #<Formulary::FormulaNamespaceAPIe0d511356bd44120af49cc96c9dcf3b3::Gcc:0x0000000111ef0d00>
Did you mean?  version_scheme

### What did you expect to happen?

I would like to see the successful installation of cernlib. 

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
brew tap davidchall/hep
brew info cernlib 
brew install cernlib
```

**Comment by davidchall** [comment]
This will be addressed by #324, once that's ready to merge.

**Comment by arseniy-1995** [comment]
I have the same error:

-- Performing Test Fortran_COMPILER_SUPPORTS_-fstack-protector - Success
-- Performing Test Fortran_COMPILER_SUPPORTS_-Wno-conversion
-- Performing Test Fortran_COMPILER_SUPPORTS_-Wno-conversion - Success
-- Performing Test Fortran_COMPILER_SUPPORTS_-Wno-implicit-function-declaration
-- Performing Test Fortran_COMPILER_SUPPORTS_-Wno-implicit-function-declaration - Failed
-- Performing Test Fortran_COMPILER_SUPPORTS_-Wno-return-type
-- Performing Test Fortran_COMPILER_SUPPORTS_-Wno-return-type - Success
-- CERNLIB : CMAKE_Fortran_FLAGS= --param=ssp-buffer-size=4 -fno-backslash -ffixed-line-length-72 -std=legacy -fcommon -fno-automatic -fno-second-underscore -fno-align-commons -fno-aggressive-loop-optimizations -finit-local-zero -fallow-argument-mismatch -fno-range-check -fno-bounds-check -fbacktrace -fno-strict-aliasing -Wno-cpp -Wno-uninitialized -Wno-character-truncation -Wno-line-truncation -Wno-lto-type-mismatch -Wno-integer-division -Wno-intrinsic-shadow -Wno-argument-mismatch -Wno-aggressive-loop-optimizations -Wno-unused-label -Wno-unused-function -Wno-unused-variable -Wno-maybe-uninitialized -Wno-unused-dummy-argument -no-pie -O0 -g -fallow-invalid-boz -fstack-protector -Wno-conversion -Wno-return-type
CMake Error: Unable to read from file '../contrib/man/man1/cernlib.1': Could not open extended attribute file
CMake Error at CMakeLists.txt:615 (file):
file failed to compress:
/tmp/cernlib-20231006-34473-a3xs7r/cernlib-2023.08.14.0-free/build/cernlib.1.gz

-- Configuring incomplete, errors occurred!


Please fix it as soon as possible. I really need a library "cernlib" for work.

**Comment by davidchall** [comment]
This has now been resolved by #324. Please try again.
